### DISCUSSION TITLE
Mouse pointer problem for me

### DISCUSSION THREAD

**Discussion by abolfazlkhosravani** [pubpub-zz/ppInk#161] [Q&A]
You did a great job, thank you
Up to ppInk 1.3 alpha 2 ppInk ,v1.3.2 works fine and has no problems

**Comment by abolfazlkhosravani** [comment]
https://user-images.githubusercontent.com/47294908/143680220-6d10317b-c35a-4525-b7d1-4edb73e21fc8.mp4

**Comment by abolfazlkhosravani** [comment]
https://user-images.githubusercontent.com/47294908/143680694-12b32f98-dc5b-4c65-a370-6e24934d51f4.mp4

Up to ppInk 1.3 alpha 2 ppInk v1.3.2 works fine and has no problems, but newer and newer versions do not work well for me

**Reply by pubpub-zz** [reply]
> Video_2021-11-27_153636.mp4
> 
> Up to ppInk 1.3 alpha 2 ppInk v1.3.2 works fine and has no problems, but newer and newer versions do not work well for me

Hi @abolfazlkhosravani, Can you please clarify the issue your facing

**Reply by abolfazlkhosravani** [reply]
![2021-11-29_150002](https://user-images.githubusercontent.com/47294908/143861776-0c19dcb9-05c6-4330-a5b5-777624adc4e1.png)

When I press the Mouse pointer button, it is as if I hit the dock/Undock toolbar button

**Reply by abolfazlkhosravani** [reply]
> > Video_2021-11-27_153636.mp4
> > Up to ppInk 1.3 alpha 2 ppInk v1.3.2 works fine and has no problems, but newer and newer versions do not work well for me
> 
> Hi @abolfazlkhosravani, Can you please clarify the issue your facing

https://user-images.githubusercontent.com/47294908/143870742-32d2bf3e-4dbe-47ae-affe-856cac9a1427.mp4

This part where you select the desired desktop size with the mouse does not work for me and only the toolbar is collapsed and in default the full desktop screen is selected

**Reply by pubpub-zz** [reply]
I've tried my 'daily' beta version (available in #162) and it is working fine. can you check it on your side ?

**Reply by abolfazlkhosravani** [reply]
> I've tried my 'daily' beta version (available in #162) and it is working fine. can you check it on your side ?
Thanks, I tried this version too, I still have the same problem, I have to add a special program to my Windows?

**Reply by abolfazlkhosravani** [reply]
> I've tried my 'daily' beta version (available in #162) and it is working fine. can you check it on your side ?

https://github.com/pubpub-zz/ppInk/discussions/161#discussioncomment-1715381

**Reply by pubpub-zz** [reply]
It may more likely be that you have an other program who is changing the behavior(logger, ...)

**Comment by abolfazlkhosravani** [comment]
![2021-11-27_155517](https://user-images.githubusercontent.com/47294908/143681350-c6cfa529-db7e-4308-b1b7-b9c5a52252d5.png)
The cursor while writing is too big if it is an arrow

**Reply by pubpub-zz** [reply]
About the arrow size, you can change it just replacing the cursorarrow.ico file in ppink folder

**Reply by pubpub-zz** [reply]
@abolfazlkhosravani
have you been able to modify the image to match your needs?

**Reply by abolfazlkhosravani** [reply]
> have you been able to modify the image to match your needs?

Thankful . Yes, I changed

**Comment by abolfazlkhosravani** [comment]
![2021-11-27_155646](https://user-images.githubusercontent.com/47294908/143681382-6f256f06-20aa-452f-95bd-c97f67698f42.png)
and too small if we choose pen tip and it does not look good

**Comment by abolfazlkhosravani** [comment]
![2021-11-27_162939](https://user-images.githubusercontent.com/47294908/143682363-87f69d84-5d89-4093-a5df-cc479f4178fe.png)

I think this is the shape of a pen

**Comment by abolfazlkhosravani** [comment]
Thank you for your efforts . The cartoon is very great and beautiful. Please continue. Please do so in the update version if possible🙏🌹

**Comment by pubpub-zz** [comment]
@abolfazlkhosravani 
        It is hard to see issues that you are still having. can you please open issues on problem you are still facing. Thanks
